src/components/Footer/StyledFooter.js

```diff
@@ -12,8 +12,10 @@ const ContactPanelRow = styled(Row)``
 
 // Individual social media contact column
 const ContactPanelColumn = styled(Col)`
-  justify-content: center;
-  padding: 30px;
+${genericStyles}
+  justify-content: ${props => (props.justify ? props.justify : 'center')};
+  text-align: ${props => (props.align ? props.align : 'center')};
+  padding-bottom: 30px;
```

Comment:
`padding-bottom` could use the same props treatment since `Col` does have a padding prop